toonz/sources/toonz/filebrowsermodel.cpp

```diff
@@ -51,13 +51,10 @@ TFilePath getMyDocumentsPath() {
   return TFilePath((const char *)[documentsDirectory
       cStringUsingEncoding:NSASCIIStringEncoding]);
 #else
-  std::string path(getenv("HOME"));
-  if(path.empty()) return TFilePath();
-  path += "/Documents";
-  QString pathAsQString = QString::fromStdString(path);
-  QDir dir(pathAsQString);
+  QDir dir = QDir::home();
+  dir.cd("Documents");
   if(!dir.exists()) return TFilePath();
```

Comment:
If `dir.cd("Documents")` fails to change directory, I think dir.exists() will still be true because it's still in a valid directory.
I think this can be just written as follows:

```
QDir dir = QDir::home();
if(!dir.cd("Documents")) return TFilePath();
``` 

Same for the one below.

Reply:
@manongjohn Thanks, I've made the change as suggested.